index.bs

```diff
@@ -49,198 +49,207 @@ conversion domain.
 
 # HTML monkeypatches # {#html-monkeypatches}
 
-Rewrite the anchor element to accept the following attributes:
+Rewrite the <{a}> to accept the following attributes:
 
 <pre class="idl">
 partial interface HTMLAnchorElement {
-    [CEReactions, Reflect] attribute DOMString conversiondestination;
-    [CEReactions, Reflect] attribute DOMString impressiondata;
-    [CEReactions, Reflect] attribute DOMString reportingorigin;
-    [CEReactions, Reflect] attribute unsigned long long impressionexpiry;
+    [CEReactions, Reflect] attribute DOMString attributiondestination;
+    [CEReactions, Reflect] attribute DOMString attributionsourceeventid;
+    [CEReactions, Reflect] attribute DOMString attributionreportto;
+    [CEReactions, Reflect] attribute unsigned long long attributionexpiry;
 };
 </pre>
 
-The <dfn for="a" element-attr>conversiondestination</dfn> is the
-declared destination [=scheme-and-registrable-domain=] of the anchor for
-purposes of conversion measurement
+The <dfn for="a" element-attr>attributiondestination</dfn> is an [=url/origin=]
+which declares the intended final navigation url resulting from running
+<a spec="html">follow the hyperlink</a> with the <{a}> element.
 
-The <dfn for="a" element-attr>impressiondata</dfn> is a string
-containing information about the `impression tag` and will be supplied in the
-`conversion report`.
+Note: This origin does not need to match the exact final origin of the navigation. 
+However, this navigation and the final navigation url have to produce the same result when
+passed to [=obtain a site=].
 
-The <dfn for="a" element-attr>reportingorigin</dfn> declares the
-intended [=origin=] to send the `conversion report` for this impression.
+The <dfn for="a" element-attr>attributionsourceeventid</dfn> is a string
+containing information about the `attribution source` and will be supplied in the
+[=attribution report=].
 
-The <dfn for="a" element-attr>impressionexpiry</dfn> is the amount
-of time in milliseconds the impression should be considered for conversion
-measurement and reporting reporting.
+The <dfn for="a" element-attr>attributionreportto</dfn> declares the
+intended [=origin=] to send the [=attribution report=] for this source.
 
+The <dfn for="a" element-attr>attributionexpiry</dfn> is the amount
+of time in milliseconds the attribution source should be considered for reporting.
 
-Issue: Need monkey patches passing impression data in navigation, and a mechanism
-for validating the resulting document matches the conversiondestination.
+Issue: Need monkey patches passing attribution source in navigation, and a mechanism
+for validating the resulting document matches the attributiondestination.
 
 # Structures # {#structures}
 
-<h3 dfn-type=dfn>Impression</h3>
+<h3 dfn-type=dfn>Attribution source</h3>
 
-An impression is a [=struct=] with the following items:
+An attribution source is a [=struct=] with the following items:
```

Comment:
For a followup, it'd be nice for readers if you include a sentence for each of these structs saying what they're intended to mean. Those sentences are non-normative, so you can wrap them in `<span class=note>` if you're feeling formal, but most spec authors don't bother.

Reply:
Ack.